category.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PHPMD\Category;
+
+class Category
+{
+    public static $checks;
+}
+
+Category::$checks = array(
```

Comment:
Does PHP let you do that?

I looked it up and it recommended how I'm currently doing it.